### ISSUE TITLE
[Bug] gitee_issues_timing search error

### ISSUE THREAD

**Issue by heming6666** [X-lab2017/grimoirelab-sigils#9]
The `search` of `gitee_issues_timing` panel should be `gitee_pull_request:false` instead of `gittee_pull_request:false`. 

See [gitee_issues_timing.json#L32](https://github.com/X-lab2017/grimoirelab-sigils/blob/05125b2f23cd9ddb4d726ed82e94599df356274d/json/gitee_issues_timing.json#L32)
 ```
    "searches": [
        {
            "id": "Search:gittee_pull_request:false",
            "value": {
                "columns": [
                    "_source"
                ],
                "description": "",
                "hits": 0,
                "kibanaSavedObjectMeta": {...},
                "sort": [
                    "grimoire_creation_date",
                    "desc"
                ],
                "title": "Search:gittee_pull_request:false",
                "version": 1
            }
        }
    ],
```

Related issue: #7 .

**Comment by heming6666** [comment]
Hi @wengzhenjie , maybe you can look into this issue and submit a PR to fix it.

You can refer to [GrimoireLab 极简操作手册](https://github.com/X-lab2017/grimoirelab/wiki/GrimoireLab-Gitee-%E6%8F%92%E4%BB%B6%E6%93%8D%E4%BD%9C%E6%89%8B%E5%86%8C) to help you get start with grimoirelab-gitee quickly.

Please ping me if you have any problems, many thanks!

**Comment by wengzhenjie** [comment]
OK

> Hi @wengzhenjie , maybe you can look into this issue and submit a PR to fix it.
> 
> You can refer to [GrimoireLab 极简操作手册](https://github.com/X-lab2017/grimoirelab/wiki/GrimoireLab-Gitee-%E6%8F%92%E4%BB%B6%E6%93%8D%E4%BD%9C%E6%89%8B%E5%86%8C) to help you get start with grimoirelab-gitee quickly.
> 
> Please ping me if you have any problems, many thanks!

ok

**Comment by heming6666** [comment]
I wrote a brief documentation about how to develop grimoirelab-gitee: https://github.com/X-lab2017/grimoirelab/wiki/How-to-develop-GrimoireLab-Gitee.

@wengzhenjie You can check it out and hope it's helpful for you.

BTW, maybe you can set up a project tracker to manage all of your contribution to grimoirelab. 
Here's my example: https://github.com/heming6666/grimoirelab-project-tracker

Of course, it's optional and up to you.

**Comment by heming6666** [comment]
Fixed by #11. Thank you @wengzhenjie .
